File: legacy-expo/lib/drinks.ts
/** Preset drink types offered in the drink logger. Users can also add custom ones. */
export const PRESET_DRINKS = [
  "Beer",
  "Cocktail",
  "Wine",
  "Shot",
  "Seltzer",
  "Non-alcoholic",
] as const;

export type PresetDrink = (typeof PRESET_DRINKS)[number];

/** Emoji shown next to each preset drink. Falls back to a generic glass. */
export const DRINK_EMOJI: Record<string, string> = {
  Beer: "🍺",
  Cocktail: "🍸",
  Wine: "🍷",
  Shot: "🥃",
  Seltzer: "🥤",
  "Non-alcoholic": "🧃",
};

export function drinkEmoji(type: string): string {
  return DRINK_EMOJI[type] ?? "🍹";
}

File: NYCGayBars/Logic/Drinks.swift
import Foundation

/// Preset drink types offered in the drink logger. Users can also add custom ones.
let PRESET_DRINKS = ["Beer", "Cocktail", "Wine", "Shot", "Seltzer", "Non-alcoholic"]

private let DRINK_EMOJI: [String: String] = [
    "beer": "🍺",
    "cocktail": "🍸",
    "wine": "🍷",
    "shot": "🥃",
    "seltzer": "🥤",
    "non-alcoholic": "🧃",
]

/// Emoji shown next to a drink type (case-insensitive, so a custom-typed
/// "beer" matches the preset). Falls back to a generic glass.
func drinkEmoji(_ type: String) -> String {
    DRINK_EMOJI[type.lowercased()] ?? "🍹"
}
